src/util/freehand/FreehandHandleData.js

```diff
@@ -0,0 +1,29 @@
+/**
+* @author JamesAPetts
```

Comment:
Could we remove the `@author` tags? Git already tracks who did the work.

Reply:
No problem, I've just got into this habbit when jsdoc-ing, I'll remove them.